packages/menu/src/SubMenu/SubMenu.tsx

```diff
@@ -351,7 +349,7 @@ SubMenu.displayName = 'SubMenu';
 SubMenu.propTypes = {
   title: PropTypes.string,
   description: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
-  href: PropTypes.string,
+  href: PropTypes.oneOfType([PropTypes.string, PropTypes.any]),
```

Comment:
SubmenuProps should not have href listed explicitly, right?

Reply:
`PropTypes` doesn't like it if it's not listed explicitly

Reply:
i think that's an okay tradeoff -- don't we want the href type to be defined by Inferred?

Reply:
no I know.. I think we should remove `href` from SubMenuProps, and be okay with it not being defined in propTypes.